src/components/DashboardPage.js

```diff
@@ -1,15 +1,24 @@
 import React, { Component } from 'react';
-import { Query, Mutation } from 'react-apollo';
 import gql from 'graphql-tag';
 import _ from 'lodash';
 import logo from '../logo.png';
 import { Link } from 'react-router-dom';
 import { withAlert } from 'react-alert';
 import { getFirebaseUserInfo } from '../utils/auth';
+import AuthenticatedQuery from './AuthenticatedQuery';
 
 class DashboardPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // userShows: [],
```

Comment:
I don't think you should be storing userShows in state. Apollo should be able to provide you with everything